subsys/net/lib/nrf_cloud/src/nrf_cloud.c

```diff
@@ -465,16 +406,15 @@ static int user_data_set(const struct cloud_backend *const backend,
 	return 0;
 }
 
-static const struct cloud_api nrf_cloud_api = {
-	.init = init,
-	.uninit = uninit,
-	.connect = connect,
-	.disconnect = disconnect,
-	.send = send,
-	.ping = ping,
-	.input = input,
-	.user_data_set = user_data_set
-};
+static const struct cloud_api nrf_cloud_api = { .init = init,
+						.uninit = uninit,
+						.connect = connect,
+						.disconnect = disconnect,
+						.send = send,
+						.ping = ping,
+						.input = input,
+						.user_data_set =
+							user_data_set };
```

Comment:
Not sure I'm a big fan of this change though

Reply:
I have the same problem when I run `clang-format` locally, it messes up `enum` and `structs`.

Reply:
reverted change.